robocop/reports/file_stats_report.py

```diff
@@ -13,24 +16,61 @@ class FileStatsReport(Report):
         Processed 7 files from which 5 files contained issues.
     """
 
-    def __init__(self):
+    def __init__(self, compare_runs):
         self.name = "file_stats"
         self.description = "Prints overall statistics about number of processed files"
         self.files_count = 0
         self.files_with_issues = set()
+        super().__init__(compare_runs)
 
     def add_message(self, message: Message):
         self.files_with_issues.add(message.source)
 
-    def get_report(self) -> str:
+    def persist_result(self):
+        return {"files_count": self.files_count, "files_with_issues": len(self.files_with_issues)}
+
+    @staticmethod
+    def get_plurar(count):
```

Comment:
```suggestion
    def get_plural(count):
```

Please also fix in other places.